<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_view</class>
 <widget class="QWidget" name="main_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="lbl_date">
   <property name="geometry">
    <rect>
     <x>799</x>
     <y>0</y>
     <width>221</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>130</width>
     <height>532</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="btn_navToMain">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>128</horstretch>
        <verstretch>128</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>128</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>128</width>
        <height>128</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>СООБЩЕНИЕ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_navToDiag">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>128</horstretch>
        <verstretch>128</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>128</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>0</width>
        <height>128</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>ДИАГ.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_navToStop">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>128</horstretch>
        <verstretch>128</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>128</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>128</width>
        <height>128</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>СМЕНА</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_navToConf">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>128</horstretch>
        <verstretch>128</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>128</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>128</width>
        <height>128</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>КОНФИГ.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QcGaugeWidget" name="gauge_speed" native="true">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>90</y>
     <width>450</width>
     <height>450</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hl_gauge_speed"/>
  </widget>
  <widget class="QLabel" name="lbl_last_motioning_state">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>210</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="main_weight_view" name="svg_weight" native="true">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>90</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>100</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QLabel" name="lbl_wgt">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>25</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>0.0 Т.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="main_weight_view" name="svg_fuel" native="true">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>90</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>100</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QLabel" name="lbl_flv">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>25</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>0.0 Л.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lbl_last_loading_state">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>260</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>lbl_date</zorder>
  <zorder>gauge_speed</zorder>
  <zorder>lbl_last_motioning_state</zorder>
  <zorder>svg_weight</zorder>
  <zorder>svg_fuel</zorder>
  <zorder>lbl_last_loading_state</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QcGaugeWidget</class>
   <extends>QWidget</extends>
   <header>Utils/Widgets/QcGauge/QcGaugeWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>main_weight_view</class>
   <extends>QWidget</extends>
   <header>App/Dmp/Presentation/Views/Main/main_weight_view.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
